core/chains/evm/txm/orchestrator.go

```diff
@@ -347,6 +347,10 @@ func (o *Orchestrator[BLOCK_HASH, HEAD]) GetTransactionStatus(ctx context.Contex
 	}
 }
 
+func (o *Orchestrator[BLOCK_HASH, HEAD]) GetTransactionFee(ctx context.Context, transactionID string) (fee *commontypes.TransactionFee, err error) {
+	return
```

Comment:
Shoudl this return an error? If error is not nil, won't callers expect the fee to be non-nil?

Reply:
+1

Reply:
I was just fulfilling the interface, there are plenty calls on Orchestrator that are stubbed. The actual implementation is located in chainlink-framework

Reply:
This is a dangerous pattern that should not be perpetuated. Why not return an error?
```suggestion
	return nil, errors.New("unimplemented")
```